cli/commands/build.js

```diff
@@ -30,42 +34,60 @@ const mkdir = directory => {
   }, root);
 };
 
-function getDestinationFromConfig(source, target) {
-  const config = getPackageConfig();
+function getDestinationFromConfig(source, target, config) {
+  if (!config) {
+    throw new Error('You must pass the location of the "clio.toml" file.');
+  }
+
   const buildConfig = config.build;
+  const buildDirectory = buildConfig.directory;
 
-  if (!buildConfig) {
+  if (!buildDirectory) {
     throw new Error(
-      `No build configuration has been found. It is a "[build]" section on you "${CONFIGFILE_NAME}" file.`
+      `The build directory is missing on your "${CONFIGFILE_NAME}".\n\nExample:\n\n[build]\ndirectory = "build"\n`
     );
   }
 
-  const buildTarget =
-    target ||
-    (buildConfig.target in config.target
-      ? config.target[buildConfig.target].target
-      : buildConfig.target);
+  return path.join(source, `${buildDirectory}/${target}`);
+}
 
-  const buildDirectory = buildConfig.directory;
+// FIXME I'm not sure if this function should stay here
+function getBuildTarget(targetOverride, config) {
+  if (!config) {
+    throw new Error('You must pass the location of the "clio.toml" file.');
+  }
+  const buildConfig = config.build;
 
-  if (!buildDirectory) {
+  if (!buildConfig) {
     throw new Error(
-      `The build directory is missing on your "${CONFIGFILE_NAME}".\n\nExample:\n\n[build]\ndirectory = "build"\n`
+      `No build configuration has been found. It is a "[build]" section on you "${CONFIGFILE_NAME}" file.`
```

Comment:
Typo: _you_ should be _your_.
Suggestion: Change `It is a "[build]" section on you "${CONFIGFILE_NAME}" file.` to `Please add a [build] section to your "${CONFIGFILE_NAME}" file.`

Reply:
Fixed on https://github.com/clio-lang/clio/pull/98/commits/f187fab97d68591dde94202f3c56b9bc30b3b9d5.